app/workers/vic/delete_old_uploads.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module VIC
+  class DeleteOldUploads
+    include Sidekiq::Worker
+
+    def perform
+      delete_docs
+      delete_photos
+    end
+
+    def photos_to_keep
+      guids = []
+      ::InProgressForm.where(form_id: 'VIC').find_each do |ipf|
+        form = ipf.data_and_metadata[:form_data]
+        guids << form['photo']['confirmationCode']
```

Comment:
dd214s are not required for everyone and someone can save the form before uploading the profile photo and never come back so we should handle any of the fields possibly being nil. also since we're not using the in progress form id in the path anymore can we get rid of that code from the uploader?

Reply:
Yep. Sounds good.